lib/axios.js

```diff
@@ -99,9 +104,91 @@ function deleteLabel(exit, token, label) {
         })
 }
 
+// Gets headers from request to list all repos for an organization
+function headRepoList(exit, token, owner, host, perPage) {
+    // Variables
+    let url = 'https://' + host + '/api/v3/orgs/' + owner + '/repos?sort=full_name&per_page=' + perPage
+
+    // HEAD request
+    return axios    
+        .head(url, {
+            headers: {
+                Authorization: 'Bearer ' + token,
+                accept: 'application/vnd.github.v3+json'
+            }
+        })
+        .catch(function (err) {
+            if (err.response) {
+                if (err.response.status == 404) {
+                    echo.error('404: Not found.')
+                    echo.error('URL seems to be faulty: ' + url)
+                    echo.error('Please check your arguments and try again.', true)
+                }
+            } else {
+                echo.error('An unexpected error occurred.')
+                echo.error(err.message, exit)
+            }
+        })
+}
+
+// Gets the "pageNum" page of the repos list with up to "perPage" records
+function getReopsByPage(exit, token, owner, host, pageNum, perPage) {
```

Comment:
Grammar: `getReposByPage()`
```suggestion
function getReposByPage(exit, token, owner, host, pageNum, perPage) {
```